src/community/smart-data-loader/src/test/resources/org/geoserver/smartdataloader/visitors/meteo-parameters-gml.xsd

```diff
@@ -78,6 +78,7 @@
           <xs:element maxOccurs="1" minOccurs="0" name="name" type="xs:string" />
           <xs:element maxOccurs="1" minOccurs="0" name="surname" type="xs:string" />
           <xs:element maxOccurs="1" minOccurs="0" name="company" type="xs:string" />
+          <xs:element maxOccurs="1" minOccurs="0" name="active" type="xs:boolean" />
```

Comment:
Looks like we would need at least one tested field for every supported added postgresql type.

Reply:
fixed